tox.ini

```diff
@@ -4,54 +4,64 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27-test, py37-test, py27-flake8, docs
+envlist =
+    py27-{test}
+    py38-{test,cov,flake8}
+    docs
 
-# The exhaustive list of environments is:
-# envlist = py{27,34,35}-{test,cov}, py{27,34,35}-flake8, docs
+# exhaustive list
+# py{27,35,36,37,38}-{test,cov,flake8}, docs
+
+[gh-actions:env]
+PLATFORM =
+    ubuntu-latest: linux
+    windows-latest: windows
+
+[testenv]
+passenv =
+    LANG # avoids output errors
+    NOSE_SHOW_SKIPPED # Undocumented feature of nose-show-skipped.
+deps =
+    {test,cov}: {[_test]deps}
+    flake8: {[_flake8]deps}
+    py27: pathlib
+commands =
+    py27-test: python -m nose {posargs}
+    py38-cov: python -bb -m nose --with-coverage {posargs}
+    py38-flake8: flake8 {posargs} {[_flake8]files}
```

Comment:
As above.

Reply:
I thought the idea was that we would have the minimal defaults up here. Maybe I misunderstood. Do you mind making a suggestion and I'll commit it?